sdk/communication/Azure.Communication.CallAutomation/tests/SessionRecordedEvents/CallDialogAutomatedLiveTests/ConsecutiveDialogsTest.json

```diff
@@ -0,0 +1,199 @@
+{
+  "entries": [
+    {
+      "body": "[{\u0022id\u0022:\u0022Sanitized\u0022,\u0022source\u0022:\u0022calling/callConnections/431fbf00-0211-48d6-aa17-9b9ab3491975\u0022,\u0022type\u0022:\u0022Microsoft.Communication.CallDisconnected\u0022,\u0022data\u0022:{\u0022callConnectionId\u0022:\u0022431fbf00-0211-48d6-aa17-9b9ab3491975\u0022,\u0022serverCallId\u0022:\u0022aHR0cHM6Ly9hcGkuZmxpZ2h0cHJveHkuc2t5cGUuY29tL2FwaS92Mi9jcC9jb252LXVzd2UtMDItcHJvZC1ha3MuY29udi5za3lwZS5jb20vY29udi9ndTdYVUtQNFVVQ0k3c0pGUmo2b2Z3P2k9MTAtNjAtMTMtMTYxJmU9NjM4MjUzMDg2OTY4NzgwNDY0\u0022,\u0022correlationId\u0022:\u002277270897-fc79-4fae-82a7-599f9710eead\u0022},\u0022time\u0022:\u00222023-07-20T19:47:48.4605406\\u002B00:00\u0022,\u0022specversion\u0022:\u00221.0\u0022,\u0022datacontenttype\u0022:\u0022application/json\u0022,\u0022subject\u0022:\u0022calling/callConnections/431fbf00-0211-48d6-aa17-9b9ab3491975\u0022}]",
```

Comment:
CallConnectionId, ServerCallId, and correlationId are fake values, or do they need to be sanitized?

Reply:
They don't need to be sanitized - everything is run on a resource specifically marked for testing